mods/common/languages/en.ftl

```diff
@@ -165,7 +165,7 @@ unrevealed-terrain = Unrevealed Terrain
 ## ServerlistLogic, GameInfoStatsLogic, ObserverShroudSelectorLogic, SpawnSelectorTooltipLogic
 team-no-team =
     { $team ->
-        [zero] No Team
+        [0] No Team
```

Comment:
we use `zero` everywhere

Reply:
See https://github.com/OpenRA/OpenRA/pull/20230#issuecomment-1230165304. I'm not sure why this doesn't work here. I found `[0]` being used for `player-disconnected` as well.

Reply:
`[zero]` should be correct, see (https://projectfluent.org/fluent/guide/selectors.html)

Reply:
As discussed on Discord: `[zero]` is not a category in English, so we're using `[0]` now for cases in which that represents a special case. (Using `[zero]` wasn't semantically correct anyway, see https://github.com/OpenRA/OpenRA/pull/20048#discussion_r950058482).